rq_dashboard/__init__.py

```diff
@@ -1,19 +1,2 @@
-# Modules/blueprints
-from .dashboard import dashboard
-
-
-class RQDashboard(object):
-    def __init__(self, app=None, url_prefix='/rq', auth_handler=None):
-        self.url_prefix = url_prefix
-        if app is not None:
-            self.app = app
-            self.init_app(app)
-        else:
-            self.app = None
-        self.auth_handler = auth_handler
-        self.redis_conn = None
-
-    def init_app(self, app):
-        """Initializes the RQ-Dashboard for the specified application."""
-        app.register_blueprint(dashboard, url_prefix=self.url_prefix)
-        app.extensions['rq-dashboard'] = self
+import default_settings  # NOQA
+import blueprint  # NOQA
```

Comment:
Even though we're in `0.x`, I'd avoid breaking API's wherever possible. If we're going to break it, let's do it right the first time. Doing `from rq.blueprint import blueprint` looks messy. Is there no nice way to make `from rq import blueprint` work?